.circleci/config.yml

```diff
@@ -2,24 +2,48 @@ version: 2
 jobs:
   build:
     docker:
-      - image: 218546966473.dkr.ecr.us-east-1.amazonaws.com/circle-ci:tap-tester
+      - image: 218546966473.dkr.ecr.us-east-1.amazonaws.com/circle-ci:stitch-tap-tester
     steps:
       - checkout
       - run:
           name: 'Setup virtual env'
           command: |
             python3 -mvenv /usr/local/share/virtualenvs/tap-iterable
             source /usr/local/share/virtualenvs/tap-iterable/bin/activate
-            pip install -U 'pip<19.2' setuptools
+            pip install -U 'pip<19.2' 'setuptools<51.0.0'
             pip install .[dev]
+            pip install pylint
       - run:
           name: 'pylint'
           command: |
             source /usr/local/share/virtualenvs/tap-iterable/bin/activate
-            make test
+            pylint tap_iterable -d C,R,W
+      # - run:
```

Comment:
No integration or unit tests are being run?

Reply:
Fix is [here](https://github.com/singer-io/tap-iterable/blob/TDL-21942-add-tap-tester-tests/.circleci/config.yml#L29-#L35)